src/main/java/fox/spiteful/forbidden/FMEventHandler.java

```diff
@@ -513,11 +513,13 @@ public void onFeelPain(LivingHurtEvent event) {
             ItemStack amulet = BaublesApi.getBaubles(player).getStackInSlot(0);
             if (amulet != null && amulet.getItem() == ForbiddenItems.subCollar) {
                 int doses = 3 * (int) event.ammount;
```

Comment:
Rather than tracking two variables, you should actually just cap it here:
```suggestion
                int doses = 3 * (int) Math.min(event.ammount, event.entityLiving.getHealth());
```

Reply:
yeah, that makes more sense